packages/sdk-snippets/src/targets/ruby/rails/webhooks/client.ts

```diff
@@ -37,9 +39,10 @@ export const rails: Client = {
     blank();
 
     push('def webhook', 1);
+    startSection('verification');
     push('# Your ReadMe secret', 2);
     push(`secret = '${secret}'`, 2);
-    push('# Verify the request is legitimate and came from ReadMe', 2);
```

Comment:
There was a weird unicode character here instead of a space.

Reply:
Very weird, i thought I fixed this. This broke one of my builds locally at some point. I think it was a zero width space character or something. No idea how it got there 🤷